.ci/Jenkinsfile

```diff
@@ -65,90 +65,7 @@ pipeline {
           setEnvVar('GO_VERSION', readFile(file: '.go-version')?.trim())
         }
       }
-    }
-    stage('Check'){
-      steps {
-        withGithubNotify(context: "Check") {
-          withMageEnv(){
-           dir("${BASE_DIR}"){
-              setEnvVar('BEAT_VERSION', sh(label: 'Get beat version', script: 'make get-version', returnStdout: true)?.trim())
-              log(level: 'INFO', text: "env.BEAT_VERSION=${env.BEAT_VERSION}")
-              cmd(label: 'check', script: 'make check-ci')
-            }
-          }
-        }
-      }
-    }
-    stage('Test') {
```

Comment:
While we migrate the check we remove additional `Test` and `Package` from Jenkins.